Tests/conf.json

```
@@ -6264,4 +6264,4 @@
         "IPv6 CIDR extraction test",
         "IPv6 extraction test"
     ]
-}
+}
```

Please revert the changes in this file.
```suggestion
}

```